app/presenters/contact_presenter.rb

```diff
@@ -50,6 +50,14 @@ def links
 
 private
 
+  def parent_content_id
+    if contact.organisation.content_id == PublishFinders::HMRC_ORGANISATION_CONTENT_ID
```

Comment:
It looks like HMRC [has child organisations](https://www.gov.uk/api/content/government/organisations/hm-revenue-customs). I don't know much about contacts: would contacts in a child org of HMRC need to show up as HMRC?

Reply:
My quick scan through - all existing contacts pages are published by HMRC and not the child organisations. I don't think we've ever defined what would be the expected behaviour. It may be a future requirement, but would be a non-trivial amount of effort and probably not worth "fixing" now.